test/Syntax/round_trip_parse_gen.swift

```diff
@@ -148,7 +148,6 @@ typealias J = inout @autoclosure () -> Int
 typealias K = (@invalidAttr Int, inout Int, __shared Int, __owned Int) -> ()
 
 @objc private typealias T<a,b> = Int
-@objc private typealias T<a,b>
```

Comment:
Why did you need to remove this test case?

Reply:
Because a `typealias` always needs to have an `= <type>` to be syntactically well-formed, and this test requires that the input source be syntactically well-formed to pass. This is no longer representable without an "unknown declaration" error due to the change to make the `= type` required by the syntax grammar.